### PR TITLE
make pipes nosplash during pipebomb construction

### PR DESCRIPTION
**Pull request by zjdtmkhzt**
## About the PR 
Have you ever constructed a pipebomb?
Have you ever splashed the contents of an entire fuel tank onto a pipe frame even though only 20 units were necessary?

If you have answered yes to question 1 but not question 2, you are a liar.

This adds the NOSPLASH flag to pipe frames after the hollowing out step, so that chems can be filled into the pipe without splashing the rest of the reagent container contents onto the pipe frame.
The flag is removed at a later stage. (This was initially in case so pipe bombs could be melted with acid or whatever, but pipe bombs are a different item entirely, so it's really just because why not.)
## Why's this needed? 
Because splashing all the stuff onto pipe frames was real annoying and also a waste of fuel if you wanted to make several pipe bombs.
## Changelog

```
(u) zjdtmkhzt:
(+) Filling a pipe with chems during pipebomb construction will no longer splash the rest of the chems onto it.
```

### CODE REVIEW

**Comment by MarkNais** [comment]
Nice MHelp-to-fix time

**Comment by makkipakki** [comment]
oh gosh yes please
I've likely wasted far more units by splashing them on the pipe frame than I actually have put inside of pipe bombs.

File: code/obj/item/grenades.dm [hunk 1]
```diff
@@ -1202,6 +1203,7 @@ PIPE BOMBS + CONSTRUCTION
 				name = "pipe bomb frame"
 
 			desc = "Two small pipes joined together, filled with welding fuel and connected with a cable. It needs some kind of ignition switch."
```
Comment (MarkNais) [inline code/obj/item/grenades.dm:1205]:
```suggestion
			desc = "Two small pipes joined together, filled with explosives and connected with a cable. It needs some kind of ignition switch."
```
more generic

Comment (zjdtmkhzt) [reply code/obj/item/grenades.dm:1205]: Yeah, that is a valid point, I saw that but didn't consider I could fix it. >.<

**Pull request merged**